File-Formats/DocIO/Working-with-Charts/Format-Axis.md

```diff
@@ -0,0 +1,218 @@
+
+//Set the horizontal (category) axis title.
+chart.PrimaryCategoryAxis.Title = "Products";
+//Set the Vertical (value) axis title.
+chart.PrimaryValueAxis.Title = "In Dollars";
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customization of Border
+
+The following code snippet illustrates how to customize the border of Horizontal and vertical category axis.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//Customize the horizontal category axis.
+chart.PrimaryCategoryAxis.Border.LinePattern = OfficeChartLinePattern.CircleDot;
+chart.PrimaryCategoryAxis.Border.LineColor = Syncfusion.Drawing.Color.Blue;
+chart.PrimaryCategoryAxis.Border.LineWeight = OfficeChartLineWeight.Hairline;
+
+//Customize the vertical category axis.
+chart.PrimaryValueAxis.Border.LinePattern = OfficeChartLinePattern.CircleDot;
+chart.PrimaryValueAxis.Border.LineColor = Syncfusion.Drawing.Color.Blue;
+chart.PrimaryValueAxis.Border.LineWeight = OfficeChartLineWeight.Hairline;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customization of Font
+
+The following code snippet illustrates how to customize the border of Horizontal and vertical category axis.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//Customize the horizontal category axis font.
+chart.PrimaryCategoryAxis.Font.Color = OfficeKnownColors.Red;
+chart.PrimaryCategoryAxis.Font.FontName = "Calibri";
+chart.PrimaryCategoryAxis.Font.Bold = true;
+chart.PrimaryCategoryAxis.Font.Size = 20;
+
+//Customize the vertical category axis font.
+chart.PrimaryValueAxis.Font.Color = OfficeKnownColors.Red;
+chart.PrimaryValueAxis.Font.FontName = "Calibri";
+chart.PrimaryValueAxis.Font.Bold = true;
+chart.PrimaryValueAxis.Font.Size = 20;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Text Angle Rotation
+
+The following code snippet illustrates how to rotate the text angle for the axis title area.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//Axis title area text angle rotation.
+chart.PrimaryValueAxis.TitleArea.TextRotationAngle = 90;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Set the Maximum value in Axis
+
+The following code snippet illustrates how to set the maximum value in the axis.
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//Maximum value in the axis
+chart.PrimaryValueAxis.MaximumValue = 14.0;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Set the Number format in Axis
+
+The following code snippet illustrates how to set the number format in the axis.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//Number format for axis
+chart.PrimaryValueAxis.NumberFormat = "0.0";
+
+{% endhighlight %}
+{% endtabs %}
+
+## Set the Category Label
+
+The following code snippet illustrates how to set the category label.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+
+//Sets category labels
+chart.PrimaryCategoryAxis.CategoryLabels = chart.ChartData[2, 1, 11, 1];
+
+{% endhighlight %}
+{% endtabs %}
+
+## Manage Gridline Visibility
+
+The following code snippet illustrates how to hide or show major and minor gridlines.
```

Comment:
The following code sample illustrates how to hide or show major and minor gridlines.